common/content/hints.js

```diff
@@ -804,6 +804,7 @@ var Hints = Module("hints", {
         this.addMode("V", "View hint source in external editor",  (elem, loc) => buffer.viewSource(loc, true));
         this.addMode("y", "Yank hint location",                   (elem, loc) => editor.setRegister(null, cleanLoc(loc), true));
         this.addMode("Y", "Yank hint description",                elem => editor.setRegister(null, elem.textContent || "", true));
+        this.addMode("X", "Download YouTube Videos",              loc => io.system('youtube-dl \"' + loc + '\"'));
```

Comment:
Is it possible to add this under plugins/contrib instead to avoid putting this in everyone's config?